tests/domain/user_emails_test.py

```diff
@@ -34,7 +36,8 @@ def when_feature_send_emails_enabled_sends_a_reset_password_email_to_user(self,
         assert data['Subject'] == 'Réinitialisation de votre mot de passe'
         assert data['To'] == 'bobby@test.com'
 
-    def when_feature_send_emails_disabled_sends_email_to_pass_culture_dev(self, app):
+    @patch('utils.mailing.feature_send_mail_to_users_enabled', return_value=False)
+    def when_feature_send_emails_disabled_sends_email_to_pass_culture_dev(self, feature_send_mail_to_users_enabled,app):
         # given
         user = create_user(public_name='bobby', email='bobby@test.com', reset_password_token='AZ45KNB99H')
         mocked_send_email = Mock()
```

Comment:
```suggestion
        return_value = Mock(return_value=200)
        mocked_send_email = Mock(return_value=return_value)
```
On s'aperçoit que la variable `return_value` ne veut rien dire donc à renommer.
Je pense même que l'on peut faire ça en une seule ligne :
`mocked_send_email = Mock(return_value=200)` ?

Reply:
au final j'ai même mieux, tous les test continuent de passer sans les return_value
donc j'ai plus que cette ligne :
mocked_send_email = Mock()